recipes/clinod/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -eu -o pipefail
+
+outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $outdir
+mkdir -p $PREFIX/bin
+cp -R * $outdir/
+cp $RECIPE_DIR/clinod.py $outdir/clinod
+ls -l $outdir
+ln -s $outdir/clinod $PREFIX/bin
+chmod 0755 "${PREFIX}/bin/clinod"
+
+# clinod expects the batchman binary from SNNS in its folder:
+ln -s "`which batchman`" $outdir/batchman
```

Comment:
Other than the test file thing, this is the bit of the recipe I am least confident in.

How would you suggest I work out where Conda would install ``batchman`` from the ``snns`` package (which I have deliberately listed as both a build and runtime dependency)?

I'm probably making a number of wrong assumptions in how the Conda ecosystem works - while this passes the tests it may not work "in the wild". I guess we'd find out once the package goes live (although I'm not immediately ready to test this with my Galaxy wrapper for Effective T3).